libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -263,6 +263,14 @@ singleton AP_Baro method get_pressure float
 singleton AP_Baro method get_temperature float
 singleton AP_Baro method get_external_temperature float
 
+include AP_OpticalFlow/OpticalFlow.h
```

Comment:
```suggestion
include AP_OpticalFlow/OpticalFlow.h depends defined(OPTFLOW) && OPTFLOW == 1
```
I'm not %100 that define is correct on all vehicles, but there should be one that works.

Reply:
As mentioned below, I don't think this method can work because the definition is in the vehicle layer.